### ISSUE TITLE
Allure reporter issues while running with devtools service 

### ISSUE THREAD

**Issue by dimkin-eu** [webdriverio/webdriverio#5366]
**Environment (please complete the following information):**
 - **WebdriverIO version:**  6.1.8
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:**  sync 
 - **Node.js version:** v12.16.1
 - **NPM version:** 6.13.4
 - **Browser name and version:**  Chrome 81
 - **Platform name and version:** macOS Catalina 10.15.4 (19E287)

**package.json**
"devtool" config
```
{
  "name": "wdio6",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "wdio"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@wdio/cli": "^6.1.8",
    "@wdio/allure-reporter": "^6.1.5",
    "@wdio/devtools-service": "^6.1.8",
    "@wdio/dot-reporter": "^6.1.5",
    "@wdio/local-runner": "^6.1.8",
    "@wdio/mocha-framework": "^6.1.8",
    "@wdio/sync": "^6.1.8"
  }
}
```
"chromedriver" config ( note only changes - devtools-service replaced by 2 chrome options)

```
{
  "name": "wdio6",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "wdio"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@wdio/cli": "^6.1.8",
    "@wdio/allure-reporter": "^6.1.5",
    "chromedriver": "81.0.0",
    "wdio-chromedriver-service": "^6.0.3",
    "@wdio/dot-reporter": "^6.1.5",
    "@wdio/local-runner": "^6.1.8",
    "@wdio/mocha-framework": "^6.1.8",
    "@wdio/sync": "^6.1.8"
  }
}
```

**Config of WebdriverIO**
only changes here -    service    `["devtools"]` and `["chromedriver"]` accordingly

```
exports.config = {
    runner: "local",
    maxInstances: 10,
    logLevel: "info",
    bail: 0,
    waitforTimeout: 10000,
    connectionRetryTimeout: 90000,
    connectionRetryCount: 3,

    specs: [
        "./test/**/*Spec.js"
    ],
    capabilities: [{
        "browserName": "chrome",
        "goog:chromeOptions": {
            excludeSwitches: ["enable-automation"]
        }
    }],
    services: [
        ["devtools"]
    ],
    framework: "mocha",
    reporters: ["dot",
        ["allure", {
            outputDir: "allure-results",
            disableWebdriverStepsReporting: false,
            disableWebdriverScreenshotsReporting: false
        }]
    ],    mochaOpts: {
        ui: "bdd",
        timeout: 60000
    },
    afterTest(test, context, { error, result, duration, passed, retries }) {
        if (!passed) {
            browser.takeScreenshot();
        }
    }
};
```

Test
```
const assert = require('assert')

describe('on webdriver.io page', () => {
    it('this test will fail on purpose', () => {
        browser.url('https://webdriver.io')
        browser.takeScreenshot();
        const title = browser.getTitle()
        assert.strictEqual(title, 'Web3verIO · Next-gen browser automation test framework for Node.js')
    })
})
```

**Describe the bug**
while running using `["chromedriver"]` service 
- 2 screenshots were created and added to allure report
- all GET and POST requests to the browser are logged
- the browser has no version in the report
resulting xml report https://pastebin.com/efjRQG8X

while running using`["devtools"]` service 
- no screenshots were created and added to allure report
- no GET and POST requests to the browser are logged
- the browser has a weird version in the report - Chrome-19.4.0
resulting xml report https://pastebin.com/Y6Zvbs0E

**To Reproduce**
npm test

**Expected behavior**
to have same behaviour using devtools service as in chromedriver

**Comment by christian-bromann** [comment]
It seems that the `devtools` package doesn't log the appropriate information that the Allure reporter listens to.

- afaik there is a event thrown when a screenshot is being taken, this event seems to not being throws in [takeScreenshot.js](https://github.com/webdriverio/webdriverio/blob/master/packages/devtools/src/commands/takeScreenshot.js)
- not GET and POST requests are being logged because there are none, the Allure reporter should adjust for this and instead listen to other value information
- the devtools package is getting the version from the [user agent](https://github.com/webdriverio/webdriverio/blob/master/packages/devtools/src/index.js#L35) it seems that something is from with that logic

These are all great areas for contributions. Any help is highly appreciated.

**Comment by christian-bromann** [comment]
fixed in #5543
